rustls/src/quic.rs

```diff
@@ -98,26 +98,6 @@ mod connection {
                 Self::Server(conn) => conn.alert(),
             }
         }
-
-        /// Returns an object that can derive key material from the agreed connection secrets.
-        ///
-        /// See [RFC5705][] for more details on what this is for.
-        ///
-        /// This function can be called at most once per connection.
-        ///
-        /// This function will error:
-        ///
-        /// - if called prior to the handshake completing; (check with
-        ///   [`CommonState::is_handshaking`] first).
-        /// - if called more than once per connection.
-        ///
-        /// [RFC5705]: https://datatracker.ietf.org/doc/html/rfc5705
-        pub fn exporter(&mut self) -> Result<KeyingMaterialExporter, Error> {
-            match self {
-                Self::Client(conn) => conn.core.exporter(),
-                Self::Server(conn) => conn.core.exporter(),
-            }
-        }
```

Comment:
I am generally preferring not to have functions like this, when they are trivial to provide externally and not commonly used.

Reply:
Maybe we should get rid of more of the wrappers over client + server or at least their methods? We also now have the potential option of abstracting over them...